projects/storefrontapp/tsconfig.app.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "module": "esnext",
+    "module": "es2020",
```

Comment:
I'm thinking about being a bit more conservative here, to be as close as possible customer's implementation, what do you think? 
Is es2020 widely established already?

Reply:
Yes, it's already recommended when upgrading to angular 10:
https://angular.io/guide/migration-update-module-and-target-compiler-options